features/apimgt/org.wso2.carbon.apimgt.core.feature/pom.xml

```diff
@@ -350,11 +350,11 @@
                 </executions>
             </plugin>
         </plugins>
-        <resources>
-            <resource>
-                <directory>${project.build.directory}/docs/</directory>
-            </resource>
-        </resources>
+        <!--<resources>-->
+            <!--<resource>-->
+                <!--<directory>${project.build.directory}/docs/</directory>-->
+            <!--</resource>-->
+        <!--</resources>-->
     </build>
```

Comment:
After adding this config, none of the resources specified in the p2.inf does not get copy to the pack. We need to look into this further